### ISSUE TITLE
Log gradients with subset of training set

### ISSUE THREAD

**Issue by amritpurshotam** [amritpurshotam/computer-vision-papers#12]
Test whether wandb uses the full training set when logging the gradients. If so and it's very slow (takes approx 1hr to even read all the data from disk), then setup another dataset that only uses a smaller subset (maybe 5000 random images so hopefully at least one image from each class is selected)
